setup.py

```diff
@@ -53,173 +62,185 @@ class UploadCommand(BaseCommand):
         self._run(
-            "Running pytest…",
+            "Installing test dependencies ...",
+            [sys.executable, "-m", "pip", "install"] + validate_dependencies,
+        )
+        self._run("Running black ...", [sys.executable, "-m", "black", f"{here}/slack"])
+        self._run(
+            "Running flake8 ...", [sys.executable, "-m", "flake8", f"{here}/slack"]
+        )
+
+        target = self.test_target.replace("tests/", "")
+        self._run(
+            "Running unit tests ...",
             [
                 sys.executable,
                 "-m",
                 "pytest",
                 "--cov-report=xml",
                 f"--cov={here}/slack",
-                f"integration_tests/{target}",
+                f"tests/{target}",
             ],
         )
 
 
-class RunAllTestsCommand(ValidateCommand):
-    """Support setup.py run_all_tests"""
+class IntegrationTestsCommand(BaseCommand):
+    """Support setup.py run_integration_tests"""
 
-    description = ValidateCommand.description + "\nRun integration tests (pytest)."
+    description = "Run integration tests (pytest)."
 
     user_options = [
-        ('unit-test-target=', 'i', 'tests/{unit-test-target}'),
-        ('utt=', 'i', 'tests/{utt}'),
-        ('integration-test-target=', 'i', 'integration_tests/{integration-test-target}'),
-        ('itt=', 'i', 'integration_tests/{itt}'),
-        ('test-target=', 'i', 'integration_tests/{test-target}')
+        ("test-target=", "i", "integration_tests/{test-target}"),
+        ("legacy=", "i", "1"),
     ]
 
     def initialize_options(self):
-        self.unit_test_target = ""
-        self.utt = ""
-        self.integration_test_target = ""
-        self.itt = ""
         self.test_target = ""
+        self.legacy = ""
 
     def run(self):
-        ValidateCommand.run(self)
-        target = (self.itt or self.integration_test_target or self.test_target).replace("integration_tests/", "")
+        target = self.test_target.replace("integration_tests/", "")
+        path = (
+            f"integration_tests_legacy/{target}"
+            if self.legacy
+            else f"integration_tests/{target}"
+        )
         self._run(
-            "Running pytest…",
+            "Running integration tests ...",
             [
                 sys.executable,
                 "-m",
                 "pytest",
                 "--cov-report=xml",
                 f"--cov={here}/slack",
-                f"integration_tests/{target}",
+                path,
             ],
         )
 
 
 setup(
-    name="slackclient",
+    name="slack_sdk",
     version=__version__,
-    description="Slack API clients for Web API and RTM API",
+    description="The Slack API Platform SDK for Python",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/slackapi/python-slackclient",
+    url="https://github.com/slackapi/python-slackclient",  # TODO: update the URL
```

Comment:
With a change like this, when does this URL get updated?

Reply:
Since a patch release after we rename the GitHub repository. But again, this is not urgent as I already mentioned above.

>We can rename this when we change the GitHub repository name (in the short-term, GitHub redirects visitors for us)